paddle/fluid/framework/details/threaded_ssa_graph_executor.cc

```diff
@@ -84,6 +87,7 @@ FeedFetchList ThreadedSSAGraphExecutor::Run(
   // Clean run context
   run_op_futures_.clear();
   exception_holder_.Clear();
+  event.reset(nullptr);
```

Comment:
Why here use a smart ptr but in scope_buffered_ssa_graph_executor.cc we use a stack value?

Reply:
Because I want to stop tracking in the middle of the scope here.